conventions/documenting_code.md

```diff
@@ -115,6 +115,63 @@ class Unicorn
 end
 ```
 
+### Inheriting Documentation
+
+When using inheritance, if an instance method is only documented on the parent type; the method on the child type will inherit the documentation of the parent with `Description copied from class/struct ParentType` prepended.
```

Comment:
"When using inheritance" is unnecessary. This applies to every type, even for implicit inheritance from `Object`.

My suggestion: `When an instance method has no doc comment but a method with the same signature exists in a parent type, the documentation is inherited from that method.`